website/content/gateway/api_reference/resources/debits/recipient_information.md

```diff
@@ -23,6 +23,11 @@ Example: "Doe Jane A." (last name, first name, optional middle initial).
 {{% regex_optional %}}Required.{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}recipient[account_type] {{% regex %}}(other|rtn_and_ban|iban|pan|email|phone_number|ban_and_bic|wallet_id|social_network_id){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}The recipient's account type, i.e. the type of the account.
```

Comment:
I would say "wallet ID" isn't an account type; "wallet" is. So the choice of `account_type` sounds a bit off to me, unless e.g. `wallet_id` is changed to `wallet`.

I assume the point of the parameter to specify the type of the account reference in `recipient[account_number]`. If a wallet ID is written in `recipient[account_number]` then `wallet_id` must be chosen here. So perhaps `recipient[account_number_type]` could make sense.

I might of course be completely mistaken.

Reply:
Good point. I think I originally thought `account_type` would be better and still convey the correct meaning but I agree that `wallet_id` with `account_type` does not. I think what you are suggesting is better and I'll name it `account_number_type`